Migrations/20220704221200_AddNamesAndContactToUser.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Practice.Migrations
+{
+    public partial class AddNamesAndContactToUser : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "FirstName",
+                table: "Users",
+                type: "TEXT",
+                maxLength: 20,
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "LastName",
+                table: "Users",
+                type: "TEXT",
+                maxLength: 20,
+                nullable: true);
+
+            migrationBuilder.CreateTable(
+                name: "Contacts",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "INTEGER", nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    Phone = table.Column<string>(type: "TEXT", maxLength: 11, nullable: false),
+                    Address = table.Column<string>(type: "TEXT", maxLength: 75, nullable: false),
+                    UserUsername = table.Column<string>(type: "TEXT", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Contacts", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_Contacts_Users_UserUsername",
```

Comment:
Another practice: Please add `Id column` (set identity) for User table and use this field for the Foreign Key instead of `Username`. In real case we usually use the Guid/Int/Long/DateTime2 type as Primary key.